src/test/resources/features/Section2/Clearance/ClearanceJourney.feature

```diff
@@ -0,0 +1,176 @@
+@Regression
+Feature: Section2 Clearance Prelodged
+
+  Background:
+    Given I clear data in cache
+  #The initial section 2 Journey starts with below combination
+  #isEIDR=NO, #isExporter=NO, isExporterEORI=NO, holdTheContract=NO, carrierEoriNumber=No
+
+  @Section2 @ClearancePrelodgedJourney
```

Comment:
we were already using tags @ClrDec for clearance declaration can we use the same thing as it is shorter tag. also we will not run prelodged and arrived seperatly we can use one tag for clearance. I found similar thing across other spec.  for standard we are using @StdDec, @ClrDec, @OcaDec, @SimDec, @SupDec these tags are used across section1, section3, section 4

Reply:
Done